### PR TITLE
Website: Responsive grade band course pages

### PR DESCRIPTION
**Pull request by tanyaparker**
**/elementary-school before**
image

---

**/elementary-school after**
image

---

**/middle-school before**
image

---

**/middle-school after**
image

---

**/high-school before**
image

---

**/high-school after**
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
cc @breville

File: pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: CS Curriculum for grades 6-8
 nav: educate_nav
+video_player: true
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md:4]: How did it work without this?

**Comment by breville** [comment]
Good thing I never ask for screenshots :)

File: pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md (hunk 1 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md:4]: It didn't

**Comment by tanyaparker** [comment]
I legit thought about it but the differences are so minimal. I could include a few I guess...

**Comment by breville** [comment]
Improvements look good!

**Pull request merged**